modules/ROOT/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -76,7 +78,7 @@ sudo -u www-data php occ \
     --value '<language code>'
 ....
 
-[NOTE] 
+[NOTE]
```

Comment:
Any reason for committing a whitespace change?

Reply:
No, that's my editors work :D

I switched to Atom, maybe it will be better now. 

Or someone will tell me the option to check,